### ISSUE TITLE
think about removing solidus initial condition

### ISSUE THREAD

**Issue by jdannberg** [geodynamics/aspect#1528]
There is an initial temperature model called "solidus" that reads in a file to be used as the initial temperature and adds some spherical harmonic perturbation. Initially, it was used to generate an initial temperature model for early Mars after the freezing of the magma ocean.

In principal, we have the same functionality in other plugins, and the name of this plugin is misleading now that we have melt migration and melting models that could compute a solidus that is consistent with the material model (which is also something that would be useful to have as a temperature initial condition). 

We should discuss what to do with this file and where to move it best if we want to remove it from the main code (test? cookbooks/future?). 

@SiqiZhang: You created this model, do you have an opinion on this? Did you use this file for some of your models and would be willing to write a cookbook for it? Otherwise, I would vote for moving it to a test.

**Comment by SiqiZhang** [comment]
I agree that the current state of this plugin may not be so useful for other modelers. I have no objection of moving it to a test.
However, we could improve it to use mantle adiabatic and limit it with mantle solidus (mainly for the upper mantle) get from material model. This may make it more useful for other mantle models.
